bin/agent_runner.rb

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env ruby
+
+# This process is used to maintain Huginn's upkeep behavior, automatically running scheduled Agents and
+# periodically propagating and expiring Events. It also running TwitterStreamAgents and Agents that support long running
+# background jobs.
+
+Dotenv.load if Rails.env == 'development'
```

Comment:
I think it's totally reasonable to keep, if we can make it work both with and without a Procfile.